### PR TITLE
Some builder registry details

### PR DESCRIPTION
**Pull request by ckartik**

### CODE REVIEW

File: get-started/relays.mdx [hunk 1]
```diff
@@ -3,11 +3,88 @@ title: Relay Guide
...
+Opting into mev-commit as a relay means that you sign up to deliver mev-boost bids only from builder addresses that have registered with mev-commit. This is necessary as it acts as a sybil resistance mechanism, disallowing block builders from making commitments but building a block under a different builder key that mev-commit would not be able to penalize.
 
-Relays need to look out for mev-commit opted-in validator addresses to watch for proposers, and filter block builder bids to addresses that are registered with mev-commit when it’s that proposer’s turn. See [Querying for Proposers](/get-started/providers/Querying-for-proposers) to see how you can check whether an upcoming proposer is opted in to mev-commit or not.
+Information about the providers (the name for entities like builders on our network) is stored in our provider registry. You can interact with the registry at the contract address at {ProviderRegistryAddress}.
```
Comment (shaspitz) [inline get-started/relays.mdx:13]:
```suggestion
Information about the providers (entities like builders who commit on our network) is stored in our provider registry. You can interact with the registry at the contract address at {ProviderRegistryAddress}. 
```

File: get-started/relays.mdx [hunk 2]
```diff
@@ -3,11 +3,88 @@ title: Relay Guide
...
\ No newline at end of file
+    ```javascript
+    // Import the ethers library
+    const ethers = require("ethers");
+
+    // Define the provider using the RPC URL
+    const provider = new ethers.JsonRpcProvider("https://chainrpc.testnet.mev-commit.xyz/");
+
+    // Define the contract address and ABI
+    const contractAddress = "0x5960774AD41D03DAB4916a30bD2190f8b3b3b4b2";
```
Comment (shaspitz) [inline get-started/relays.mdx:26]: nit: makes it more clear which contract we're talking about
```suggestion
    const providerRegistryAddress = "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707";
```

File: get-started/relays.mdx [hunk 3]
```diff
@@ -3,11 +3,88 @@ title: Relay Guide
...
+    const abi = [
+        "event ProviderRegistered(address indexed provider, uint256 stakedAmount)"
+    ];
+
+    // Create a new contract instance
+    const contract = new ethers.Contract(contractAddress, abi, provider);
```
Comment (shaspitz) [inline get-started/relays.mdx:32]:
```suggestion
    const contract = new ethers.Contract(providerRegistryAddress, abi, provider);
```

File: get-started/relays.mdx [hunk 4]
```diff
@@ -3,11 +3,88 @@ title: Relay Guide
 sidebarTitle: Relays
 icon: 'arrow-right-arrow-left'
 ---
+import { ProviderRegistryAddress } from '/snippets/testnet-addresses.mdx';
+export const ProviderRegistryExplorerUrl = `http://explorer.testnet.mev-commit.xyz/address/${ProviderRegistryAddress}`;
```
Comment (shaspitz) [inline get-started/relays.mdx:7]: Isn't this var already defined in snippets/testnet-addresses.mdx?

File: get-started/relays.mdx [hunk 5]
```diff
@@ -3,11 +3,88 @@ title: Relay Guide
...
+    '0x028677f17372b4B19C49Cfa74A92654d18eDfe49',
+    '0x051f91FeC4421bfDC5A4504E123ABb73C7790Ed1',
+    '0x762bba6B80dcFaf65f9F91A6a6ed60559f4F003c',
+    ...
+    ]
+    ```
+  </Accordion>
+
+  <Accordion title="Additional Information" icon="info">
+    Check out the [Provider Registry Explorer](http://explorer.testnet.mev-commit.xyz/address/0x5960774AD41D03DAB4916a30bD2190f8b3b3b4b2) for more details on registered providers.
```
Comment (shaspitz) [inline get-started/relays.mdx:83]: We should use the `ProviderRegistryExplorerUrl` var here right?

**Review by shaspitz — APPROVED** [review]
LGTM! just some small things

**Comment by shaspitz** [comment]
Also this PR would be useful since we're introducing a new place that contract addresses are hardcoded https://github.com/primev/mev-commit/pull/86. Best solution here imo is https://github.com/primev/mev-commit/issues/45

File: get-started/relays.mdx (hunk 5 above)
Comment (ckartik) [reply get-started/relays.mdx:83]: I literally spent an hour trying to get that to work on mvintlify. Not wasting more time on it. Will go ahead and delete it for now. We can explore this kind of code organization later.

File: get-started/relays.mdx [hunk 6]
```diff
@@ -3,11 +3,87 @@ title: Relay Guide
...
+  </Accordion>
+
+</AccordionGroup>
+
+Relays need to look out for mev-commit opted-in validator addresses to watch for proposers, and filter block builder bids to addresses that are registered with mev-commit when it's that proposer's turn. See [Querying for Proposers](/get-started/providers/Querying-for-proposers) to see how you can check whether an upcoming proposer is opted in to mev-commit or not.
```
Comment (shaspitz) [inline get-started/relays.mdx:87]:
```suggestion
Relays need to look out for mev-commit opted-in proposer addresses, and only deliver block builder bids from builder addresses that are registered with mev-commit when it's that proposer's turn. See [Querying for Proposers](/get-started/providers/Querying-for-proposers) to see how you can check whether an upcoming proposer is opted in to mev-commit or not.
```

**Pull request merged**